encoding/toml.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { deepAssign } from "../util/deep_assign.ts";
 import { pad } from "../strings/pad.ts";
+import { format, EOL } from "../fs/eol.ts";
```

Comment:
This seems unnecessary https://deno.land/style_guide.html#withindeno_stdminimizedependenciesdonotmakecircularimports

Reply:
edited.